dev-python/h5py/h5py-2.10.0.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1 flag-o-matic
+
+DESCRIPTION="Simple Python interface to HDF5 files"
+HOMEPAGE="http://www.h5py.org/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc test examples mpi"
+
+RDEPEND="
+	sci-libs/hdf5:=[mpi=,hl(+)]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]"
+
+DEPEND="
+	${RDEPEND}
+	dev-python/cython[${PYTHON_USEDEP}]
+	dev-python/pkgconfig[${PYTHON_USEDEP}]
+	doc? (
+		dev-python/alabaster[${PYTHON_USEDEP}]
+		>=dev-python/sphinx-1.3.1[${PYTHON_USEDEP}]
```

Comment:
This is **most likely** a BDEPEND and you don't need PYTHON_USEDEP for it then. Also check how alabaster is needed (you can leave PYTHON_USEDEP for that).

EDIT: you can maybe use `distutils_enable_sphinx` here, but I don't have time to check this ebuild throughly and confirm it. I have a strong feeling it should work.

Reply:
`distutils_enable_sphinx` works but only with `--no-autodoc`